scripts/codemod/helpers/codemodHelpers.js

```diff
@@ -0,0 +1,284 @@
+// @flow strict
```

Comment:
Splitting out these helpers into their own files will make it easier for us to identify dead code in the future. "Kitchen sink" util files like this tend to grow endlessly…also, we're already in a `/helpers` directory, so it's a bit strange to keep them all in one file